hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/TokenMintHandlerTest.java

```diff
@@ -134,8 +134,8 @@ void acceptsValidNonFungibleTokenMintTxn() {
 
         // number of owned NFTs should increase
         assertThat(writableAccountStore.get(treasuryId).numberOwnedNfts()).isEqualTo(4);
-        // treasury relation supply will not increase since its not fungible token change
-        assertThat(writableTokenStore.get(nonFungibleTokenId).totalSupply()).isEqualTo(1000L);
+        // token total supply should be increased
+        assertThat(writableTokenStore.get(nonFungibleTokenId).totalSupply()).isEqualTo(1002L);
```

Comment:
@Neeharika-Sompalli is this change correct?

Reply:
yes, seems correct. Found a bug. Thanks @georgi-l95